src/arche/rules/coverage.py

```diff
@@ -98,3 +100,39 @@ def compare_scraped_fields(source_df: pd.DataFrame, target_df: pd.DataFrame) ->
         result.add_info(f"New - {', '.join(new_fields)}")
 
     return result
+
+
+def anomalies(jobs_keys: List[str]) -> Result:
```

Comment:
Wondering if the API wouldn't be better if
`anomalies(target="", sample=[""])`.

Also, not sure if `target` is a good name because in `Arche`, `target` is the previous job, not the current one.
So, maybe `source`..